README.md

```diff
@@ -1,3 +1,7 @@
+| readme        | reference                               | faq                           |
+|      ---      |                  ---                    |            ---                |
+| this document | [docs/reference.md](./docs/reference.md)| [docs/faq.md](./docs/faq.md)  |
```

Comment:
Just one row please with the links having the text on this row. More compact and half the reading.

`| readme | [reference]() | [faq]() |`